spec/Dockerfile.puppetdb

```diff
@@ -1,7 +1,8 @@
-FROM puppet/puppetdb:6.13.1
+FROM puppet/puppetdb
 
 # Use our own certs so this doesn't have to wait for puppetserver startup
 COPY fixtures/ssl/ca.pem /opt/puppetlabs/server/data/puppetdb/certs/certs/ca.pem
-COPY fixtures/ssl/cert.pem /opt/puppetlabs/server/data/puppetdb/certs/certs/pdb.pem
+COPY fixtures/ssl/cert.pem /opt/puppetlabs/server/data/puppetdb/certs/certs/server.crt
```

Comment:
Do these fixtures need to be renamed to avoid confusion?

Reply:
It looks like they're reused in a couple locations, so I don't think there's a need to.